conftest.py

```diff
@@ -91,6 +92,14 @@ def reset_clickhouse_tables():
         sync_execute(item)
 
 
+def run_additional_setup_ops():
```

Comment:
aren't async migrations ee only?

Reply:
Well this ee stuff is changing and the answer is actually that going forward, no, they won't be ee-only. But anyway, the CH setup happens in both places so good to add for consistency